play28-json-tests-sc14/src/main/scala/play/api/libs/json/scalacheck/JsValueGenerators.scala

```diff
@@ -0,0 +1,163 @@
+package play.api.libs.json.scalacheck
+
+import java.math.MathContext
+
+import org.scalacheck.Arbitrary._
+import org.scalacheck.Gen._
+import org.scalacheck.Shrink._
+import org.scalacheck.{Arbitrary, Gen, Shrink}
+import play.api.libs.json._
+
+import scala.language.implicitConversions
+
+trait JsValueGenerators {
```

Comment:
Make a `class` for better forwards compatibility or only allow importing from an `object`

Reply:
This would break places that extend this trait for the implicits. This is a problem for v5.